### PR TITLE
Replace `standard library` with `Ballerina library` text

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Replace `standard library` with `Ballerina library` text.
> Fixes #

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: community/newsletter/2022-6.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ permalink: /community/newsletter/2022-6/
...
 
-						<p style="font-family: 'Roboto', Helvetica,sans-serif;font-size: 14px;color: rgba(0,0,0,0.87);line-height: 21px;font-weight: 400;text-align: left;margin-top: 10px;"><a class="linkbody" href="https://ballerina.io/downloads/?utm_source=mailer&amp;utm_medium=email&amp;utm_campaign=mailer_email_ballerina_nwsltr_july22_220728" style="color: #20b6af;text-decoration: underline;" target="_blank">Swan Lake 2201.1.1</a> is the first patch release of Ballerina 2201.1.0 (Swan Lake Update 1) and includes a new set of bug fixes to the compiler, runtime, standard library, and developer tooling.</p>
+						<p style="font-family: 'Roboto', Helvetica,sans-serif;font-size: 14px;color: rgba(0,0,0,0.87);line-height: 21px;font-weight: 400;text-align: left;margin-top: 10px;"><a class="linkbody" href="https://ballerina.io/downloads/?utm_source=mailer&amp;utm_medium=email&amp;utm_campaign=mailer_email_ballerina_nwsltr_july22_220728" style="color: #20b6af;text-decoration: underline;" target="_blank">Swan Lake 2201.1.1</a> is the first patch release of Ballerina 2201.1.0 (Swan Lake Update 1) and includes a new set of bug fixes to the compiler, runtime, Ballerina library, and developer tooling.</p>
```
Comment (shafreenAnfar) [inline community/newsletter/2022-6.md:45]: Do we need to change the old ones?

Comment (shafreenAnfar) [reply community/newsletter/2022-6.md:45]: I think this question is relevant to many files I guess.

Comment (praneesha) [reply community/newsletter/2022-6.md:45]: @keizer619 @anupama-pathirage

Comment (anupama-pathirage) [reply community/newsletter/2022-6.md:45]: Yes, Agreed with @shafreenAnfar here. We don't need to change old places like newsletters, published blogs etc. They are already sent over mail/published (ppl have seen those already) This is a new terminology we are using and we need to update all learn pages/home page/api docs or whatever current content only.

File: components/ai/code/ai-bbe/transcribe-speech-or-music-with-whisper.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
 ---
 title: 'Transcribe speech or music with Whisper'
-description: "OpenAI’s Whisper API makes speech and music computable! Easily transcribe speech or music in any language into text. Ballerina’s powerful standard library makes manipulating audio files and processing the results trivially simple."
+description: "OpenAI’s Whisper API makes speech and music computable! Easily transcribe speech or music in any language into text. Ballerina’s powerful Ballerina library makes manipulating audio files and processing the results trivially simple."
```
Comment (anupama-pathirage) [inline components/ai/code/ai-bbe/transcribe-speech-or-music-with-whisper.md:3]: It is not meaningful to say `Ballerina’s powerful Ballerina library` as Ballerina is repeating weirdly here.

File: components/integration/use-cases/UseCases.js [hunk 3]
```diff
@@ -56,7 +56,7 @@ export default function UseCases(props) {
                                 <div className={styles.cardDescription}>
                                     <p>Automation is just code after all - write a main() and do whatever you want.</p>
 
-                                    <p>Use Ballerina&apos;s extended standard library to connect to any system, speak any protocol, process any data, and run it anywhere (on a VM, in Kubernetes, or just as a script).</p>
+                                    <p>Use Ballerina&apos;s extended Ballerina library to connect to any system, speak any protocol, process any data, and run it anywhere (on a VM, in Kubernetes, or just as a script).</p>
```
Comment (anupama-pathirage) [inline components/integration/use-cases/UseCases.js:59]: incorrect to say `extended Ballerina library`

File: downloads/0.9.x-release-notes/0.970.0/RELEASE_NOTE.html [hunk 4]
```diff
@@ -424,8 +424,8 @@ <h2>Messaging endpoints</h2>
 }
 </code></pre>
 
-<h1>Standard Library</h1>
-<p>The Ballerina standard library provides a set of commonly used functionalities. The following packages are available as a part of the standard library:</p>
+<h1>Ballerina library</h1>
```
Comment (anupama-pathirage) [inline downloads/0.9.x-release-notes/0.970.0/RELEASE_NOTE.html:427]: Avoid release notes, blogs, etc

File: public/spec/auth/spec.md [hunk 5]
```diff
@@ -7,7 +7,7 @@ _Updated_: 2022/02/17
 _Edition_: Swan Lake  
 
 ## Introduction
-This is the specification for the Auth standard library of [Ballerina language](https://ballerina.io/), which is used for authorization of listeners and clients (HTTP, gRPC, GraphQL, WebSocket, WebSub, etc.).
+This is the specification for the Auth Ballerina library of [Ballerina language](https://ballerina.io/), which is used for authorization of listeners and clients (HTTP, gRPC, GraphQL, WebSocket, WebSub, etc.).
```
Comment (anupama-pathirage) [inline public/spec/auth/spec.md:10]: It is incorrect to say Library for single package. 

Package is the sharable unit and library is a collection of such packages. So it is incorrect to say `Auth Ballerina library`. So instead something like `Auth package of Ballerina Library` or something will do.

@praneesha / @shafreenAnfar / @daneshk / @ThisaruGuruge

File: swan-lake/development-tutorials/build-a-graalvm-executable/build-the-executable-locally.md [hunk 6]
```diff
@@ -199,7 +199,7 @@ Now, you tested a simple Ballerina HTTP server application for GraalVM compatibi
 
 ## Create GraalVM-compatible library packages
 
-Packages that solely use Ballerina standard libraries and connectors or those that use platform Java libraries provided by the distribution are inferred to be compatible with GraalVM.
+Packages that solely use Ballerina libraries and connectors or those that use platform Java libraries provided by the distribution are inferred to be compatible with GraalVM.
```
Comment (anupama-pathirage) [inline swan-lake/development-tutorials/build-a-graalvm-executable/build-the-executable-locally.md:202]: It is incorrect to Ballerina Libraries. It is a singular thing. Refer to https://github.com/ballerina-platform/ballerina-dev-website/pull/7782#discussion_r1328880832

File: swan-lake/other/tooling-guide/intellij-idea-plugin/set-up-intellij-idea/use-intellij-plugin-features 2.md [hunk 7]
```diff
@@ -149,7 +149,7 @@ You expand/collapse the following Ballerina code segments using the icons in the
 
 ## Go to definition
 
-This option allows you to view the definition of a selected variable, function, an object etc., within the same file, in a separate file, in the same module, or in a file of a different module, of the same project or of the [Standard Library](/learn/api-docs/ballerina/).
+This option allows you to view the definition of a selected variable, function, an object etc., within the same file, in a separate file, in the same module, or in a file of a different module, of the same project or of the [Ballerina library](/learn/api-docs/ballerina/).
```
Comment (anupama-pathirage) [inline swan-lake/other/tooling-guide/intellij-idea-plugin/set-up-intellij-idea/use-intellij-plugin-features 2.md:152]: Is this a valid link. anyway not correct to point API docs of `ballerina` org only as Ballerina library should contain API docs of ballerina and ballerianx orgs.

**Comment by anupama-pathirage** [comment]
@praneesha Please recheck all the files. 

In the previous terminology we had 3 things mainted by Ballerina platform. 

- Lang library (ballerina/lang.*) 
- Standard Library (ballerina/*)
- extended library (ballerinax/*)

But from external developer point of view no need to separate those into different categories. So the new terminology is we use generic term `Ballerina library` for all the 3 things above.

Also be cautious on the correct terminology for the term `library`.

Package is the sharable unit, and library is a collection of such packages.  So some of the following things are incorrect.
- `Auth Ballerina library` is incorrect as we don't say `library` for a single package. Instead, something like `Auth package of Ballerina Library` or something is correct.
- `Ballerina libraries` is incorrect. It is a singular thing. We have many packages and all of them are within one library. Analogy is `Books` and `Library`. There are many `books` in a single library.
-

File: components/ai/code/ai-bbe/create-images-with-dall-e.md [hunk 8]
```diff
@@ -1,4 +1,4 @@
 ---
 title: 'Create images with DALL-E and Stable Diffusion'
-description: "Stable Diffusion and OpenAI’s DALL-E image APIs generate or edit images with text based instructions. Ballerina’s powerful standard library makes uploading, downloading and processing images a breeze."
+description: "Stable Diffusion and OpenAI’s DALL-E image APIs generate or edit images with text based instructions. Ballerina’s powerful Ballerina library makes uploading, downloading and processing images a breeze."
```
[the remark this answers is not in the document]
Comment (praneesha) [reply components/ai/code/ai-bbe/create-images-with-dall-e.md:3]: Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/7834/files.

File: downloads/0.9.x-release-notes/0.970.0/RELEASE_NOTE.html (hunk 4 above)
Comment (praneesha) [reply downloads/0.9.x-release-notes/0.970.0/RELEASE_NOTE.html]: Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/7834/files.

File: components/ai/code/ai-bbe/transcribe-speech-or-music-with-whisper.md (hunk 2 above)
Comment (praneesha) [reply components/ai/code/ai-bbe/transcribe-speech-or-music-with-whisper.md:3]: Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/7834/files.

File: swan-lake/other/tooling-guide/intellij-idea-plugin/set-up-intellij-idea/use-intellij-plugin-features 2.md (hunk 7 above)
Comment (praneesha) [reply swan-lake/other/tooling-guide/intellij-idea-plugin/set-up-intellij-idea/use-intellij-plugin-features 2.md:152]: Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/7834/files.

File: swan-lake/development-tutorials/java-interoperability/java-interoperability-guide/the-bindgen-tool.md [hunk 9]
```diff
@@ -222,7 +222,7 @@ In that way, users can populate null values to the underneath Java API by provid
 the generated functions will return nil values in case the counterpart Java method returns null.
 
 ## Support for Java casting
-The `ballerina/jballerina.java` module of the Ballerina standard library provides the `cast` function to support Java casting. This could be used to cast Ballerina binding classes into their subtypes based on assignability.
+The `ballerina/jballerina.java` module of the Ballerina library provides the `cast` function to support Java casting. This could be used to cast Ballerina binding classes into their subtypes based on assignability.
```
[the remark this answers is not in the document]
Comment (praneesha) [reply swan-lake/development-tutorials/java-interoperability/java-interoperability-guide/the-bindgen-tool.md:225]: Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/7834/files.

File: swan-lake/development-tutorials/build-a-graalvm-executable/build-the-executable-locally.md (hunk 6 above)
Comment (praneesha) [reply swan-lake/development-tutorials/build-a-graalvm-executable/build-the-executable-locally.md:202]: Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/7834.

File: components/integration/use-cases/UseCases.js (hunk 3 above)
Comment (praneesha) [reply components/integration/use-cases/UseCases.js:59]: Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/7834/.

File: public/spec/auth/spec.md (hunk 5 above)
Comment (praneesha) [reply public/spec/auth/spec.md]: Will be fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/7834/.

File: swan-lake/development-tutorials/build-a-graalvm-executable/graalvm-executable-overview.md [hunk 10]
```diff
@@ -36,7 +36,7 @@ The only downside is that the GraalVM native image build is a highly complicated
 
 From Ballerina 2201.7.0 (SwanLake) onwards, Ballerina supports GraalVM AOT compilation to generate standalone executables by passing the `graalvm` flag in the build command: `bal build --graalvm`. The generated executable contains the modules in the current package, their dependencies, Ballerina runtime, and statically linked native code from the JDK.
 
-Ballerina runtime, [standard libraries](/learn/ballerina-specifications/#standard-library-specifications), and the Ballerina extended modules are GraalVM-compatible. Therefore packages developed only using these libraries are also GraalVM-compatible. Furthermore, Ballerina reports warnings when the GraalVM build is executed for a project with GraalVM-incompatible packages.
+Ballerina runtime, [Ballerina libraries](/learn/ballerina-specifications/#standard-library-specifications), and the Ballerina extended modules are GraalVM-compatible. Therefore packages developed only using these libraries are also GraalVM-compatible. Furthermore, Ballerina reports warnings when the GraalVM build is executed for a project with GraalVM-incompatible packages.
```
[the remark this answers is not in the document]
Comment (praneesha) [reply swan-lake/development-tutorials/build-a-graalvm-executable/graalvm-executable-overview.md:39]: Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/7834/.

**Comment by praneesha** [comment]
Closing as the above will be addressed in https://github.com/ballerina-platform/ballerina-dev-website/pull/7834/.

**Pull request closed without merging**